opendg/_storage/base.py

```diff
@@ -30,14 +28,6 @@ def slice_nodes(self, nodes: List[int]) -> 'DGStorageBase':
     def get_nbrs(self, nodes: List[int]) -> Dict[int, List[Tuple[int, int]]]:
         r"""Return a list of neighbour, timestamp pairs for each node in the nodes list."""
 
-    @abstractmethod
```

Comment:
It's not immediately clear that we will need these as part of the generic API, since _laziness_ is an implementation detail more than anything. For instance, the current naive dictionary implementation is automatically materialized. Hence, I removed these for the time being.